docs/configure/extensions/extension-marketplace.md

```diff
@@ -373,11 +373,15 @@ Get more information about [extension runtime security](/docs/configure/extensio
 
 The Visual Studio Marketplace signs all extensions when they are published. VS Code verifies this signature when you install an extension to check the integrity and the source of the extension package.
 
-When you install an extension, you might see the following error message: `Cannot install extension because Visual Studio Code cannot verify the extension signature`.
+When you install an extension, you might see the following error message: `Cannot install extension because Visual Studio Code cannot verify the extension signature`. This error can be caused by a variety of reasons and should you encounter this error, exercise caution before deciding to install anyway. If you want to disable extension signature verification, you can use the `setting(extensions.verifySignature)` setting.
```

Comment:
```suggestion
> [!IMPORTANT]
> When you install an extension, you might see the following error message: `Cannot install extension because Visual Studio Code cannot verify the extension signature`. This error can be caused by a variety of reasons and should you encounter this error, exercise caution before deciding to install anyway. Disable extension signature verification with the `setting(extensions.verifySignature)` setting.
```